lib/fp.js

```diff
@@ -174,6 +174,26 @@ const restLeft = fn => (...spreadArgs) => {
   fn(args, ...namedArgs, callback);
 };
 
+// Identity function
+//   x - any argument
+// Result: argument x
+const id = x => x;
+
+// Async identity function
+//   x - any argument
+//   callback - function, (err, data)
+const asyncId = (x, callback) => {
+  callback(null, x);
+};
+
+// Mapable object iteration identity function
+//   items - array
+//   fn - function
+//   callback - function, (err, data)
+const mapIterationId = (items, fn, callback) => {
```

Comment:
I see no reason behind this function.

Reply:
ping. What is the reason behind this function?

Reply:
Perhaps it would be convenient to return this function where the iterator function is expected, however we have nothing to give

Reply:
We discussed this and agreed to remove function